### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by AshutoshM29**
Added wishlist page for the e-com website

### CODE REVIEW

File: Pages/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    z-index: 2;
+}
+
+.img-content-horizontal {
+    display: flex;
+    font-size: 2rem;
+    color: var(--white);
+    z-index: 3;
+    position: absolute;
+    margin-top: 0.6rem;
```
Comment (rajaranjanmr) [inline Pages/wishlist.css:36]: suggestion : if value is less than 10px you can give in px

File: Pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Urban Tails</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+        integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://uiclean.netlify.app/Components/global.css" />
```
Comment (rajaranjanmr) [inline Pages/wishlist.html:12]: you can use either this or the local ones -- here is code redundant

File: Pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+    <link rel="stylesheet" href="/styles.css">
```
Comment (rajaranjanmr) [inline Pages/wishlist.html:13]: when you are using global.css -- then please import style.css also in that file

File: Pages/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+            <li class="list-item">
+                <a href="https://twitter.com/Astomi_29?s=08"> <i class="icons-social fab fa-twitter"></i></a></li>
+            <li class="list-item">
+                <a href="https://www.instagram.com/the.astomi/"> <i class="icons-social fab fa-instagram"></i></a></li>
+            <li class="list-item">
+                <a href="https://www.linkedin.com/mwlite/in/ashutosh-mishra-b50423206"><i
+                        class="icons-social fab fa-linkedin"></i></a></li>
+
+        </ul>
+        <div class="with-love">Made with ❤️ by : Ashutosh</div>
```
Comment (rajaranjanmr) [inline Pages/wishlist.html:258]: suggestion : you can use icon here for more flexibility if needed

**Review by rajaranjanmr — APPROVED** [review]

**Review by piyushsh25 — APPROVED** [review]
Code is well structured and clean. You can merge.

File: Pages/wishlist.html (hunk 4 above)
Comment (AshutoshM29) [reply Pages/wishlist.html:258]: ok, I'll keep that in mind

**Pull request merged**